basic-assignments-5-start/basic-assignments-5-start/src/app/movie-card-component/movie-card-component.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component } from '@angular/core';
+import { FILMS } from '../star-wars-fake-db/film-data';
+
+@Component({
+  selector: 'app-movie-card-component',
+  templateUrl: './movie-card-component.component.html',
+  styleUrls: ['./movie-card-component.component.scss'],
+})
+export class MovieCardComponentComponent {
+  movies = FILMS;
+  
+  title = 'basics-assignments-5-start';
+  opening_crawl = `Luke Skywalker has returned to
```

Comment:
The best practice is always to define the type of the variable and use camelCase instead